docker/Dockerfile

```diff
@@ -47,6 +47,8 @@ RUN pip3 install pybullet stable-baselines3 && \
     apt update && \
     DEBIAN_FRONTEND=noninteractive apt install -y \
     python3-pygame \
+    ros-${ROS_DISTRO}-rosserial-arduino \
+    ros-${ROS_DISTRO}-rosserial \
```

Comment:
This might be redundant since `rosserial-arduino` should install `rosserial`. Not sure if we really care though, since the command will still work just fine.